### DISCUSSION TITLE
Deleting elements from a page

### DISCUSSION THREAD

**Discussion by htgoebel** [zauberzeug/nicegui#904] [Q&A] [answered]
### Question

Hi,
I wonder whether it's possible to delete elements from a page — not just hide them.

Specifically, I wonder whether the Tryton web-client (see https://demo.tryton.org/) could be implemented in nicegui. In Tryton, the content of each tab-panel is created dynamically based on information from the server. If the content-instructions have changed, after closing and re-opening a tab, the new content needs to be show. Thus the WUI element needs to be deleted from the web page.

Any idea or example how this could be done?

(I managed to create tab panels on event, so this question is only about deleting.)
Specifically, I'd like to add and remove tab-panels from the page. (I managed to add tab

**Comment by falkoschindler** [comment, marked as answer]
Each element has a `remove` method. See here for an example: https://nicegui.io/documentation#clear_containers

**Reply by htgoebel** [reply]
Thx, I overlooked it.

**Reply by davetapley** [reply]
Looks like this has moved to https://nicegui.io/documentation/section_page_layout#clear_containers ?
